2.7.0/done/result_releng.md

```diff
@@ -0,0 +1,171 @@
+
+## releng
+### bc breaking
+
+- Use Manylinux 2.28 and CXX11_ABI=1 for following all released Linux wheel builds (#143423) (##146200) (#148028) (#148135) (#148195) (#148129)
```

Comment:
Can you include a description of what is before and after in terms of what a user may expect? E.g., answer how would this affect someone?